### DISCUSSION TITLE
yarn start Failure

### DISCUSSION THREAD

**Discussion by HZT3434** [penrose/penrose#938] [Q&A] [answered]
I have followed the guide on Wiki to prepare the env for penrose and unfortunately I always receive the warning **_Parsing error: Cannot read file '\penrose\tsconfig.json'_**  in the file 'packages\core\src\index.ts' . I cannot find the file in the folder and it keeps screaming. How to fix it?

**Comment by wodeni** [comment, marked as answer]
Are you getting this error on the command line or in the IDE? If you are using VSCode, we recommend that you use our workspace file: https://github.com/penrose/penrose/blob/main/penrose.code-workspace. Some IDEs don't have great support for monorepos and might fail to find the `tsconfig` file. Instead, each package in `packages` has their own `tsconfig`, and our workspace config will help VSCode resolve them.

**Comment by HZT3434** [comment]
Hi. Thank you for the kind response and just like you said I am using vscode and the program failed to find the tsconfig file. I will try the solution. 
Have a good day and again, thank you for creating such a great program.


Regards,
Huang 
















At 2022-04-25 22:44:44, "wodeni" ***@***.***> wrote:

Are you getting this error on the command line or in the IDE? If you are using VSCode, we recommend that you use our workspace file: https://github.com/penrose/penrose/blob/main/penrose.code-workspace. Some IDEs don't have great support for monorepos and might fail to find the tsconfig file. Instead, each package in packages has their own tsconfig, and our workspace config will help VSCode resolve them.
